src/analysis/calculus/deriv.lean

```diff
@@ -127,12 +127,10 @@ by simp [has_deriv_at, has_deriv_at_filter, has_fderiv_at]
 
 lemma deriv_within_zero_of_not_differentiable_within_at
   (h : ¬ differentiable_within_at 𝕜 f s x) : deriv_within f s x = 0 :=
-have ¬ ∃ f', has_deriv_within_at f f' s x, by contrapose! h; rcases h with ⟨f', h⟩; exact ⟨_, h⟩,
-by simp [deriv_within, this]
+by unfold deriv_within; rw fderiv_within_zero_of_not_differentiable_within_at; simp *
```

Comment:
Please use the `by { foo, bar, xyzzy }` style. Also elsewhere.

Reply:
Done.